tools/android_x86_toolchain.cmake

```diff
@@ -1,3 +1,4 @@
+set(ANDROID TRUE)
```

Comment:
Please remove this line from this file and other files:
ANDROID is already set to 1:)
https://cmake.org/cmake/help/latest/variable/ANDROID.html

Reply:
reasonable